### ISSUE TITLE
Retro Achievements functionality 

### ISSUE THREAD

**Issue by stuffu** [TapiocaFox/Daijishou#466]
I can’t get retro achievements to work in Daijisho, but I might have the wrong idea how it’s implemented and supposed to work.

To use achievements, I have only entered credentials in Daijisho and enabled it. Achievements in retroarch is disabled.

in the retro achievements widget I see the correct info, are the credentials just for that widget? I thought that it also tracked progress in game, but that’s maybe not the case?

**Comment by TapiocaFox** [comment]
Yes. Did you disabled auto platforms updating? Otherwise, make sure that the RetroAchevements fields in the platform settings is correctly setup.

**Comment by TapiocaFox** [comment]
Here is a screenshot for example.

![Screenshot_20230605-034127](https://github.com/TapiocaFox/Daijishou/assets/22817012/901f44f2-49c0-4fef-a825-651e7099c290)

**Comment by stuffu** [comment]
Mine looks exactly the same in game boy, Super Nintendo has SNES. They are untouched from setup. auto platforms on. Would really like to get this working :)

**Comment by Jetup13** [comment]
I'm assuming your mentioning that Retroachievements isn't working while playing games? Have you enabled achievements within RetroArch?

> RetroArch > Settings > Achievements 

From there you can sign in and enable Retroachievements. If your game matches a link rom a notification will pop-up in game.

**Comment by stuffu** [comment]
> I'm assuming your mentioning that Retroachievements isn't working while playing games? Have you enabled achievements within RetroArch?
> 
I had but suddenly I got credential errors and disabled it. Wasn’t quite sure if daijisho was relying on RA achievements or had its own integration. But I guess I also need achievements in RA enabled then…

**Comment by Jetup13** [comment]
Daijishou's Retroachievements implementation doesn't relay anything towards emulators. It's more of a quick view of the website. 

I'll suggest making sure your on the latest version of RetroArch:

* [Stable](https://buildbot.libretro.com/stable/1.15.0/android/)
* [Nightly](https://buildbot.libretro.com/nightly/android/)

Make sure your account information is correct within RetroArch. 

If that doesn't work try changing your Retroachievements password. I've heard in the past others having problems when logging in with special characters in their passwords.
